docs/how-to/spacetimedb-standalone.md

```diff
@@ -0,0 +1,250 @@
+# Self Hosting SpacetimeDB
+
+This tutorial will guide you through setting up SpacetimeDB on an Ubuntu 24.04 server, securing it with HTTPS using Nginx and Let's Encrypt, and configuring a systemd service to keep it running.
+
+## Prerequisites
+- A fresh Ubuntu 24.04 server (VM or cloud instance of your choice)
+- A domain name (e.g., `example.com`)
+- `sudo` privileges on the server
+
+Also, join the community! We have an official discord server: https://discord.gg/spacetimedb
+
+## Step 1: Create a Dedicated User for SpacetimeDB
+For security purposes, create a dedicated `spacetimedb` user to run SpacetimeDB with a home directory at `/stdb`:
+
+```sh
+sudo mkdir /stdb
+sudo useradd --system spacetimedb
+sudo chown -R spacetimedb:spacetimedb /stdb
+```
```

Comment:
Does this actually set the user's homedir? I think you have to set `-d` or `--home` to `useradd` to do that. It's not clear to me that you actually want to set the user's homedir to be `/stdb`; I think you just want to create `/stdb` as the SpacetimeDB data dir. If that's the case, you shouldn't use the phrase "home directory" to describe this.

Reply:
Oh sorry for the confusion. I actually rewrote this section to not set the home directory, I will fix the text above.